backend/de.metas.business/src/main/java/de/metas/order/paymentschedule/PaySchedule.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * #%L
+ * de.metas.business
+ * %%
+ * Copyright (C) 2025 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.order.paymentschedule;
+
+import de.metas.util.lang.Percent;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import org.jetbrains.annotations.Nullable;
+
+@Value
+@Builder
+public class PaySchedule
+{
+	@NonNull PayScheduleId id;
+	@NonNull Percent discount;
+	@NonNull Percent percentage;
+
+	@Nullable String netDay;
```

Comment:
what is 
* String netDay -> if day of week, consider using java.time.DayOfWeek
* int netDays